src/client/extension.ts

```diff
@@ -105,6 +107,10 @@ export async function activate(context: ExtensionContext): Promise<IExtensionApi
     const lintingEngine = serviceManager.get<ILintingEngine>(ILintingEngine);
     lintingEngine.linkJupiterExtension(jupyterExtension).ignoreErrors();
 
+    // Activate the data science features
```

Comment:
I forgot to push your changes before I did mine so my local branch has both of our changes in it.